meta/main.yml

```diff
@@ -23,3 +23,6 @@ galaxy_info:
   - system
   - networking
 dependencies: []
+collections:
+  - ansible.posix
+  - community.crypto
```

Comment:
What does this mean?

Reply:
Ansible has moved most modules/filters/etc into collections. Each collection has a namespace, its name. These two contain the `authorized_key` and `openssl_publickey` modules respectively. 

These need to be installed using `ansible-galaxy collection` (from the `devel` version`) and can be found under `https://github.com/ansible-collections/`.

If the `collections` key is not used in `meta/main.yml`, each module will need to have the full namespace in every task in the role. E.g.

```
- name: task
  ansible.posix.authorized_key:
    ...
```

For playbooks, there is also a `collections` key but this has no effect on roles.